tests/kafkatest/tests/consumer_test.py

```diff
@@ -212,31 +156,28 @@ def test_consumer_bounce(self, clean_shutdown, bounce_mode):
 
     @matrix(clean_shutdown=[True, False], enable_autocommit=[True, False])
     def test_consumer_failure(self, clean_shutdown, enable_autocommit):
-        partition = TopicPartition(self.STOPIC, 0)
+        partition = TopicPartition(self.TOPIC, 0)
         
-        consumer = self._setup_consumer(self.STOPIC, enable_autocommit=enable_autocommit)
-        producer = self._setup_producer(self.STOPIC)
+        consumer = self.setup_consumer(self.TOPIC, enable_autocommit=enable_autocommit)
+        producer = self.setup_producer(self.TOPIC)
 
         consumer.start()
-        self._await_all_members(consumer)
+        self.await_all_members(consumer)
 
         partition_owner = consumer.owner(partition)
         assert partition_owner is not None
 
         # startup the producer and ensure that some records have been written
         producer.start()
-        wait_until(lambda: producer.num_acked > 1000, timeout_sec=10,
-                   err_msg="Producer failed waiting for messages to be written")
+        self.await_produced_messages(producer)
 
         # stop the partition owner and await its shutdown
         consumer.kill_node(partition_owner, clean_shutdown=clean_shutdown)
         wait_until(lambda: len(consumer.joined_nodes()) == (self.num_consumers - 1) and consumer.owner(partition) != None,
-                   timeout_sec=self.session_timeout+5, err_msg="Timed out waiting for consumer to close")
+                   timeout_sec=self.session_timeout_sec+5, err_msg="Timed out waiting for consumer to close")
```

Comment:
Was this previously a bug that the timeout was incorrectly using the ms version of the timeout?